versions/2.x/models/Lease.json

```diff
@@ -1,6 +1,6 @@
 {
    "type":"Lease",
-   "derivedFrom": "https://schema.org/Thing",
+   "derivedFrom": "#Thing",
```

Comment:
@thill-odi Should this be `derivedFrom` still rather than `subClassOf` now it's inheriting from an OA model?